validateImages.js

```diff
@@ -2,57 +2,69 @@ const fs = require('fs');
 const maxFileSize = 100 * 1024; // 100KB in bytes
 
-fs.readdir(dataFolderPath, (err, files) => {
-  if (err) {
-    console.error('Error reading files:', err);
-    process.exit(1);
-  }
-
-  files.forEach((file) => {
-    console.log(`running validation check on ${file}...`);
-
-    // check if image is svg or png
-    const extension = file.slice(-4);
-    if (!allowedExtensions.includes(extension)) {
-      console.error(`\nInvalid file type: ${file}`);
-      console.error(`Allowed extensions: ${allowedExtensions.join(', ')}`);
+function validateImages(dataFolderPath) {
+  fs.readdir(dataFolderPath, (err, files) => {
+    if (err) {
+      console.error('Error reading files:', err);
       process.exit(1);
     }
 
-    const filePath = join(dataFolderPath, file);
+    for (let file in files) {
+      if (excludedFiles.includes(files[file])) {
+        continue
+      }
+      console.log(`running validation check on ${files[file]}...`);
 
-    // check if image is lesser than file max size
-    const fileSize = fs.statSync(filePath).size;
-    if (fileSize > maxFileSize) {
-      console.error(`\nFile size exceeds the limit: ${file}`);
-      console.error(`File size: ${fileSize} bytes`);
-      process.exit(1);
-    }
+      // check if image is svg or png
+      const extension = files[file].slice(-4);
+      if (!allowedExtensions.includes(extension)) {
+        console.error(`\nInvalid file type: ${files[file]}`);
+        console.error(`Allowed extensions: ${allowedExtensions.join(', ')}`);
+        // process.exit(1);
+      }
 
-    try {
-      const dimensions = sizeOf(filePath);
-      const width = dimensions.width;
-      const height = dimensions.height;
+      const filePath = join(dataFolderPath, files[file]);
 
-      if (width !== height) {
-        console.error(`\nImage is not square: ${file}`);
-        console.error(`Width: ${width}, Height: ${height}`);
+      // check if image is lesser than file max size
+      const fileSize = fs.statSync(filePath).size;
+      if (fileSize > maxFileSize) {
+        console.error(`\nFile size exceeds the limit: ${files[file]}`);
+        console.error(`File size: ${fileSize} bytes`);
         process.exit(1);
       }
 
-      if ((width < minSize || width > maxSize) && extension !== '.svg') {
-        console.error(`\nImage size is not within the allowed range: ${file}`);
-        console.error(`Width: ${width}, Height: ${height}`);
-        process.exit(1);
+      try {
+        const dimensions = sizeOf(filePath);
+        const width = dimensions.width;
+        const height = dimensions.height;
+
+        if (width !== height) {
+          console.error(`\nImage is not square: ${files[file]}`);
+          console.error(`Width: ${width}, Height: ${height}`);
+          process.exit(1);
+        }
+
+        if ((width < minSize || width > maxSize) && extension !== '.svg') {
+          console.error(`\nImage size is not within the allowed range: ${files[file]}`);
```

Comment:
Error message is a little unclear, maybe revise to:

**Image too small**
```
Image is too small: ${files[file]}.
Please make sure the image's dimensions is at least ${minSize}x${minSize}.
```

**Image too big**
```
Image is too big: ${files[file]}.
Please make sure the image's dimensions is no larger than ${maxSize}x${maxSize}.
```